pkgs/development/nim-packages/ws/default.nix

```
@@ -0,0 +1,17 @@
+{ lib, buildNimPackage, fetchFromGitHub }:
+
+buildNimPackage (final: {
+  pname = "ws";
+  version = "0.5.0";
+  src = fetchFromGitHub {
+    owner = "treeform";
+    repo = "ws";
+    rev = final.version;
+    hash = "sha256-3wVi6CjMTjcc5MJEozJN6W3TuYDb53w2MDCsv6lMH0k=";
+  };
+  preCheck = "rm tests/test_ws.nim tests/test_timeout.nim";
```

```suggestion
  preCheck = ''
    rm tests/test_ws.nim tests/test_timeout.nim
  '';
```
preCheck is normally expected to be a multiline string